canvas_modules/common-canvas/package.json

```diff
@@ -53,7 +53,7 @@
     "uuid": "^8.3.0"
   },
   "devDependencies": {
-    "@babel/core": "7.11.6",
+    "@babel/core": "^7.22.0",
```

Comment:
Can you merge the changes from main to resolve the merge conflicts? This shouldn't be needed anymore